.circleci/config.yaml

```diff
@@ -0,0 +1,138 @@
+version: 2.1
+jobs:
+  build-node14:
+    docker:
+      - image: circleci/node:14-browsers
+    working_directory: ~/repo
+    steps:
+      - run: sudo apt-get install libxss1
+      - checkout
+      - run: yarn
+      - run: yarn run ci
+      - run:
+          name: Define environment variable with lastest commit's message
+          command: |
+            echo 'export COMMIT_MESSAGE=$(git log -1 --pretty=format:"%s")' >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Lint commit message
+          command: echo "$COMMIT_MESSAGE" | npx commitlint
+      - run:
+          command: cat ./coverage/lcov.info | ./node_modules/.bin/coveralls
```

Comment:
you will need to integrate with coveralls and add relevant env variable for circleCI for this to work